device/celestica/x86_64-cel_silverstone-r0/Silverstone/port_config.ini

```diff
@@ -0,0 +1,33 @@
+# name          lanes                                    alias         index    speed
+Ethernet0       33,34,35,36,37,38,39,40                  QSFPDD1       1        400000
+Ethernet4       41,42,43,44,45,46,47,48                  QSFPDD2       2        400000
```

Comment:
This should be Ethernet8 instead of Ethernet4. We should reserve one interface per lane.

Reply:
I agree with you, I will update it.

Reply:
updated in https://github.com/Azure/sonic-buildimage/pull/5424/commits/2d79cb570370808e69015139681fe907667c81a8